wodles/aws/services/inspector.py

```diff
@@ -81,6 +97,39 @@ def send_describe_findings(self, arn_list: list):
                 self.send_msg(self.format_message(elem))
                 self.sent_events += 1
 
+    def send_describe_findings_v2(self, client, finding_arns: list):
+        """
+        Collect and send to analysisd the requested findings from Inspector v2.
+
+        Parameters
+        ----------
+        finding_arns : list[str]
+            The ARNs of the findings that should be requested to AWS and sent to analysisd.
+        """
+        if not finding_arns:
+            return
+        # Split into chunks of 10 (Inspector v2 API limit)
+        chunk_size = 10
+        aws_tools.debug(f"+++ [v2] Processing {len(finding_arns)} events", 3)
+        for i in range(0, len(finding_arns), chunk_size):
+            chunk = finding_arns[i:i + chunk_size]
+
+            try:
+                response = client.batch_get_finding_details(findingArns=chunk)
+                findings = response.get('findingDetails', [])
+
+                for finding in findings:
+                    if self.event_should_be_skipped(finding):
+                        aws_tools.debug(f'+++ The "{self.discard_regex.pattern}" regex found a match in the '
```

Comment:
These logs should also contain the `[v2]` tag to differentiate them between versions (same for v1)